bin/test_runner.rb

```diff
@@ -58,21 +71,43 @@ def is_up?
   end
 
   def get_line
-    return "#{@stage} #{@name} #{@percepts[:dup]} #{@percepts[:job_status]} #{@percepts[:destroyer_status]}"
+    return "#{@stage} #{@name} #{@percepts[:dup]} #{@percepts[:job_status]} #{@percepts[:destroyer_status]} #{@percepts[:build_id]} #{@percepts[:build]}"
   end
 
   def done?
     @stage == Done || @stage == Failed || @stage == ErrorState
   end
 
-  def update_percepts
+  def init_percepts
```

Comment:
Why do we have this function if it's only used once?  Perhaps just set these at the single location?